MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+include pyproject.toml
```

Comment:
According to https://packaging.python.org/en/latest/guides/using-manifest-in/#how-files-are-included-in-an-sdist, this should already be included by default